### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by silveroxides** [scriptscat/scriptcat#188] [Ideas]
I really like the look of the UI you use. Would it be possible for you to set up so users can translate the extension with all translatable strings in a single file and indicator to where they are located? 
The English localized user script based extensions like Tampermonkey look so ugly and outdated and I think this has great potential of it gets noticed more.

**Comment by CodFrm** [comment]
Thank you so much for your affirmation and suggestions! I'm thrilled to receive recognition from friends overseas. As for localization, it's indeed something I've been considering, but it wasn't a high priority before. However, it seems like it's time to start working on it now.

**Comment by CodFrm** [comment]
I'm delighted to inform you that we have now introduced localization support! Your input was quite valuable, and it encouraged us to prioritize this feature. Please understand, my English is not very fluent, so there may be some mistakes in the localized content. We plan to continuously refine and improve this aspect over time.

Your feedback is always welcome, and I want to thank you once again for your interest and support.

![image](https://github.com/scriptscat/scriptcat/assets/22783163/9dcf31d7-b359-4167-8a17-29b8539eecb1)

[v0.14.0-beta.1](https://github.com/scriptscat/scriptcat/releases/tag/v0.14.0-beta.1) https://github.com/scriptscat/scriptcat/pull/202

Additionally, I'd like to mention that this is a beta version. We aim to have it available on the Chrome/Edge store within the next 1-2 weeks.

**Reply by silveroxides** [reply]
That is awesome. I will take a look and see if I can contribute anything as well. So don't worry about not doing perfect translation. 

I usually install manually any way so beta is totally fine in my case

**Comment by CodFrm** [comment]
🎉 Now we have migrated the translation files to crowdin: https://crowdin.com/project/scriptcat
